src/main/java/com/mentorship/notificationservice/repository/NotificationRepository.java

```diff
@@ -0,0 +1,10 @@
+package com.mentorship.notificationservice.repository;
+
+import com.mentorship.notificationservice.domain.Notification;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.JpaSpecificationExecutor;
+
+public interface NotificationRepository extends JpaRepository<Notification, Long>,
+    JpaSpecificationExecutor<Notification> {
```

Comment:
Why do we need here JpaSpecificationExecutor?

Reply:
My mistake 
Copy/paste from other project